performance/gsheets.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+import pygsheets
+import tempfile
+
+from performance.env import check_get_env
+from performance import prod_config as config
+
+
+def get_pygsheets_client():
+    google_auth_private_key_id = os.getenv('GOOGLE_AUTH_PRIVATE_KEY_ID')
+    if not google_auth_private_key_id:
+        return pygsheets.authorize(service_file=config.GSHEETS_CREDENTIALS_FILE)
```

Comment:
Can we put a note in the config class to the effect that `GSHEETS_CREDENTIALS_FILE` will be ignored if `GOOGLE_AUTH_PRIVATE_KEY_ID` is set in the environment?